frameworks/hdfs/universe/config.json

```diff
@@ -261,6 +261,11 @@
           "description": "The HTTP port for HDFS Name Nodes. ",
           "default": 9002
         },
+        "name_node_https_port": {
```

Comment:
Could you add a unit test to HDFS' ServiceTest.java for enabling TLS and/or customizing these ports? Unit tests can now exercise universe configs. See Cassandra's ServiceTest.java for an example of enabling tls in config.json and then checking that the resulting per-task cassandra.yaml config is correctly generated:

https://github.com/mesosphere/dcos-commons/blob/2ef89c3fd66d01280ae722932d73f84d046aa712/frameworks/cassandra/src/test/java/com/mesosphere/sdk/cassandra/scheduler/ServiceTest.java#L28